src/cephadm/cephadm

```diff
@@ -3869,17 +3880,18 @@ def _get_parser():
     parser_bootstrap.add_argument(
         '--fsid',
         help='cluster FSID')
+    parser_bootstrap.add_argument(
+        '--output-dir',
+        default='/etc/ceph',
+        help='directory to write config, keyring, and pub key files')
```

Comment:
we're only using `output_dir` on the bootstrap node. woudn't it make sense to have a predictable config location also on other hosts? Feels as if we're now making the bootstrap host special by putting the config in a different location.

Reply:
Yeah, I was thinking of two things:

- option to keep cephadm (and ceph-common) pacakge installed at up to date on all hosts (regardless of root or packaged mode, although for pakcaged it required)
- option to keep ceph.conf up to date on all hosts